### DISCUSSION TITLE
Error in elastalert

### DISCUSSION THREAD

**Discussion by Firasbg50** [jertel/elastalert2#962] [Q&A] [answered]
I installed Elastalert2 with elastic8.4.1 in ubuntu 22.04 and I do a configuration to run Elastalert in the background when i restart elasticsearch it can't run and when I do "sudo journalctl -u elastalert. service" I find this error :
```
elastalert.service: Changing to the requested working directory failed: No such file or directory
sept. 17 16:56:41 elastic systemd[4245]: elastalert.service: Failed at step CHDIR spawning /usr/bin/python3: No such file or directory
```

**Comment by jertel** [comment, marked as answer]
This appears to be outside the scope of this project, unless I'm misunderstanding something. I don't recall ElastAlert 2 including a system service script.

**Reply by Firasbg50** [reply]
do you mean i should delete it ?

**Reply by jertel** [reply]
I don't know what it is. How did you install ElastAlert 2?

**Reply by Firasbg50** [reply]
with your documentation : https://elastalert2.readthedocs.io/en/latest/elastalert.html
but i did this system service script because I see someone in youtube or google (i forgot ) do this to facilitate start and stop elastalert like elasticsearch service 
![image](https://user-images.githubusercontent.com/113593198/192136823-614c96e1-3252-45db-bb35-d1e6f23bccda.png)
i create a folder in the system directory and do this config in it "windows_login_fail.yam : is my rule config "
```
[Unit]
Description= Alert System Elastalert
PartOf= elasticsearch.service





for exp this doc do this config of system service script it's in under the doc
[Service]
ExecStart=bash -c 'sleep 10; sudo python3 -m elastalert.elastalert --verbose --rule windows_login_fail.yaml'
User=root

[Install]
WantedBy= elasticsearch.service
```
for exp this doc do this config of system service in under of the doc : 
https://pawelbruski.com/posts/2021-03-16---elastic-security-siem-email-alerts-using-elastalert/

**Reply by jertel** [reply]
It looks like it can't find the config file. Why did you omit the WorkingDir statement?

**Reply by Firasbg50** [reply]
because I don't understand this statement good
